Description: Do not rename the jars to avoid confusing maven-debian-helper
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/mailapi/pom.xml
+++ b/mailapi/pom.xml
@@ -175,7 +175,7 @@
             <plugin>
                 <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
-                    <finalName>${project.artifactId}</finalName>
+                    <!--<finalName>${project.artifactId}</finalName>-->
 		    <excludes>
 			<exclude>com/sun/mail/smtp/**</exclude>
 			<exclude>com/sun/mail/imap/**</exclude>
--- a/mailapijar/pom.xml
+++ b/mailapijar/pom.xml
@@ -139,7 +139,7 @@
             <plugin>
                 <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
-                    <finalName>${project.artifactId}</finalName>
+                    <!--<finalName>${project.artifactId}</finalName>-->
 		    <excludes>
 			<exclude>com/**</exclude>
 		    </excludes>
--- a/pom.xml
+++ b/pom.xml
@@ -470,7 +470,7 @@
 	    <plugin>
 		<artifactId>maven-jar-plugin</artifactId>
 		<configuration>
-		    <finalName>${project.artifactId}</finalName>
+		    <!--<finalName>${project.artifactId}</finalName>-->
 		    <archive>
 			<!--
 			    Configure the maven-jar-plugin to pick up
